app/http.php

```diff
@@ -765,11 +770,6 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
                 'duration' => $executionTime + $coldStartTime,
             ];
 
-            // Update swoole table
-            $runtime = $activeRuntimes->get($activeRuntimeId);
-            $runtime['updated'] = \time();
-            $activeRuntimes->set($activeRuntimeId, $runtime);
-
```

Comment:
I don't think you should remove this.. This can be there as well 🆗

Reply:
what about also updating the `updated` time after the loop for runtime creation ?

Reply:
I moved code to after runtime creation. In case of no cold-start, this runs instantly.

Reply:
I put it back.